<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanelColorsHandlerWidget</class>
 <widget class="QWidget" name="PanelColorsHandlerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="fontColorLabel">
     <property name="text">
      <string>Font:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="fontColorLineEdit"/>
   </item>
   <item row="0" column="2">
    <widget class="QFrame" name="fontColorDemoFrame">
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="fontColorPushButton">
     <property name="text">
      <string>Choose</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4" rowspan="2" colspan="2">
    <layout class="QVBoxLayout" name="backgroundColorOrGradientVerticalLayout">
     <item>
      <widget class="QRadioButton" name="useBackgroundColorRadioButton">
       <property name="text">
        <string>Use background color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="useBackgroundGradientRadioButton">
       <property name="text">
        <string>Use background gradient</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="backgroundColorLabel">
     <property name="text">
      <string>Background:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="backgroundColorLineEdit"/>
   </item>
   <item row="1" column="2">
    <widget class="QFrame" name="backgroundColorDemoFrame">
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="backgroundColorPushButton">
     <property name="text">
      <string>Choose</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="backgroundGradientLabel">
     <property name="text">
      <string>Background gradient:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="backgroundGradientBaseColorLabel">
     <property name="text">
      <string>Base color:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="backgroundGradientBaseColorLineEdit"/>
   </item>
   <item row="4" column="2">
    <widget class="QFrame" name="backgroundGradientBaseColorDemoFrame">
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="backgroundGradientBaseColorPushButton">
     <property name="text">
      <string>Choose</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QPushButton" name="backgroundGradientGeneratePushButton">
     <property name="text">
      <string>Generate</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QPushButton" name="backgroundGradientAddRowPushButton">
     <property name="text">
      <string>Add row</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="backgroundGradientPreviewLabel">
     <property name="text">
      <string>Preview:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="4" colspan="2">
    <spacer name="bottomHorizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>177</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="6">
    <widget class="QTableWidget" name="backgroundGradientTableWidget"/>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QFrame" name="backgroundGradientDemoFrame">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="backgroundGradientDemoFrameTitleLabel">
        <property name="text">
         <string>Panel title</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
